src/subscription-engine.ts

```diff
@@ -12,6 +12,8 @@ import { Event } from "./subscription/Event";
 import { Notification } from "./subscription/Notification";
 import { Subscription } from "./subscription/Subscription";
 
+logger.transports[0].level = "debug";
```

Comment:
Why is this started as debug?

Reply:
This could not be here. As this feature (subscriptions) is going to be moved away from this module (soon, I hope), we might want to refactor it so that it become simpler (which will include all logging stuff).